packages/cli/src/command-reference.ts

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env node
```

Comment:
I prefer to move this code into something external, like `packages/cli/utils`, just to make it clear it's not part of the source code. Not sure if that will require some changes in the `tsdown` configuration.